empire-db/src/main/java/org/apache/empire/db/validation/DBModelChecker.java

```diff
@@ -307,6 +307,7 @@ private void checkForeignKeys(DBTable table, DBTable remoteTable, DBModelErrorHa
                 }
                 // Not found
                 handler.itemNotFound(relation);
+                break referenceLoop;

```

Comment:
Just testing integration with the dev mailing list. On the other hand not a fan of using labels in java!